server/monitor/src/main/java/org/apache/accumulo/monitor/rest/api/AllTimeTabletResults.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.accumulo.monitor.rest.api;
+
+public class AllTimeTabletResults {
+
+  public String operation;
```

Comment:
@ctubbsii and I discussed about using the `public` variable vs `private` with setters and getters, and we decided to use `public` variables, with the plus that it decreases the number of lines it takes to write each call. I was following what you had with `private` variables at first before talking with @ctubbsii, is there a reason we shouldn't go with `public` variables?